frontend/src/pages/sites/Site.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from "react";
+import { Link, useHistory } from "react-router-dom";
+import { Button } from "react-bootstrap";
+
+import { Site_findSite as Site } from "src/graphql/definitions/Site";
+import { useDeleteSite } from "src/graphql";
+import AuthContext from "src/AuthContext";
+import { isAdmin, createHref } from "src/utils";
+import { SiteLink } from "src/components/fragments";
+import DeleteButton from "src/components/deleteButton";
+import { ROUTE_SITES, ROUTE_SITE_EDIT } from "src/constants/route";
+
+interface Props {
+  site: Site;
+}
+
+const SiteComponent: React.FC<Props> = ({ site }) => {
+  const history = useHistory();
+  const auth = useContext(AuthContext);
+
+  const [deleteSite, { loading: deleting }] = useDeleteSite({
+    onCompleted: (result) => {
+      if (result) history.push(ROUTE_SITES);
+    },
+  });
+
+  const handleDelete = () => {
+    deleteSite({
+      variables: {
+        input: { id: site.id },
+      },
+    });
+  };
+
+  return (
+    <>
+      <Link to={ROUTE_SITES}>
+        <h6 className="mb-4">&larr; Site List</h6>
+      </Link>
+      <div className="d-flex no-gutters">
+        <h3 className="col-4 me-auto">
+          <SiteLink site={site} />
+        </h3>
+        {isAdmin(auth.user) && (
+          <div>
```

Comment:
This should fix the position of the Edit/Delete buttons.
(not sure about `no-gutters`, I just referenced `../categories/Category.tsx`)
```suggestion
      <div className="d-flex">
        <h3 className="me-auto">
          <SiteLink site={site} />
        </h3>
        {isAdmin(auth.user) && (
          <div className="ms-auto">
```